cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py

```diff
@@ -718,107 +432,46 @@ def test_ssa_vm_azure_windows2016_refs():
     pass
 
 
-def test_ssa_vm_scvmm2k12_multiple_vms():
+def test_ssa_vm_azure_region():
     """
-    Perform SSA on multiple VMs.
+    1. Add an Azure Instance in one region and assign it to a Resource
+    Group from another region.
+
+    Bugzilla:
+        1503295
 
     Polarion:
         assignee: sbulage
         casecomponent: SmartState
-        caseimportance: critical
-        initialEstimate: 1/4h
+        initialEstimate: 1/2h
+        startsin: 5.9
         tags: smartstate
     """
     pass
 
 
-def test_ssa_vm_scvmm2k12_centos_xfs():
+def test_ssa_vm_ec2_ubuntu_ssui():
     """
-    Add SCVMM-2012 provider.
-    Perform SSA on CentOS VM.
+    1. Provision service with Ubuntu VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
 
     Polarion:
         assignee: sbulage
         casecomponent: SmartState
-        initialEstimate: 1/2h
-        startsin: 5.6
+        caseimportance: medium
+        initialEstimate: 1/3h
+        startsin: 5.9
         tags: smartstate
     """
     pass
 
 
-def test_ssa_vm_azure_region():
+def test_ssa_vm_azure_windows2012_ssui():
```

Comment:
I would like to see a pass at parametrizing the remaining test functions here, as there are multiple azure tests on different windows versions. That should be parameters, not test function names.

For a followup PR, not this one :)

Reply:
Sure I will update it.
Thanks for the suggestion.